### DISCUSSION TITLE
Can I use `cudnn_benchmark` to check whether the A100 GPU is training efficiently compared to V100 ?

### DISCUSSION THREAD

**Discussion by Ca-ressemble-a-du-fake** [coqui-ai/TTS#1200] [General Q&A] [answered]
Hi,

I am training a model on Colab, and with a V100 I could reach step 80k in a day (around 8k / epoch), whereas with an A100 I could only reach 5k (although the batch is 4 times as big [64 vs 16]) in roughly the same time. I tried to update torch and etc as read on the internet but this does not look fast enough. Indeed to be equivalent the A100 should reach at least 20k in a day, because it is processing 4 times more samples per steps than the V100, right ? 

How can I be sure A100 is working efficiently ? Is the `cudnn_benchmark` argument in `Trainer` designed for that ?

Thanks

**Comment by lexkoro** [comment, marked as answer]
A batch of 16 samples will obv. be faster processed then a batch of 64. 
So it only makes sense that you have more steps with a batch size of 16 then with 64, in a given time window.
If you want to compare the speed you should do it with the same batch size.

**Reply by Ca-ressemble-a-du-fake** [reply]
Thank you, I'll give it a try with a batch size of 16 on A100 (when Colab gives it back to me), just to be sure A100 works properly. But with a bs of 64, it should converge quicker right ?

**Reply by lexkoro** [reply]
Yes it should need less steps to converge.
